src/sidebar/components/group-list-section.js

```diff
@@ -1,23 +1,69 @@
 'use strict';
 
-// @ngInject
-function GroupListSectionController() {
-  this.isSelectable = function(groupId) {
-    const group = this.sectionGroups.find(g => g.id === groupId);
-    return !this.disableOosGroupSelection || group.isScopedToUri;
-  };
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const GroupListItem = require('./group-list-item');
+const GroupListItemOutOfScope = require('./group-list-item-out-of-scope');
+
+/**
+ * A labeled section of the groups list.
+ */
+function GroupListSection({
+  analytics,
+  disableOosGroupSelection,
+  heading,
+  sectionGroups,
+  store,
+}) {
+  const isSelectable = groupId =>
+    !disableOosGroupSelection ||
+    sectionGroups.find(g => g.id === groupId).isScopedToUri;
+
+  return (
+    <Fragment>
+      <h2 className="group-list-section__heading">{heading}</h2>
+      <ul className="group-list-section__content">
+        {sectionGroups.map(group => (
+          <li
+            className="dropdown-menu__row dropdown-menu__row--no-border dropdown-menu__row--unpadded"
+            key={group.id}
+          >
+            {isSelectable(group.id) ? (
+              <GroupListItem
+                className="group-list-item"
+                group={group}
+                analytics={analytics}
+                store={store}
+              />
+            ) : (
+              <GroupListItemOutOfScope
+                className="group-list-item-out-of-scope"
+                group={group}
+                analytics={analytics}
+              />
+            )}
+          </li>
+        ))}
+      </ul>
+    </Fragment>
+  );
 }
 
-module.exports = {
-  controller: GroupListSectionController,
-  controllerAs: 'vm',
-  bindings: {
-    /* The list of groups to be displayed in the group list section. */
-    sectionGroups: '<',
-    /* The string name of the group list section. */
-    heading: '<',
-    /* A boolean indicating whether out of scope group selection should be disabled. */
-    disableOosGroupSelection: '<',
-  },
-  template: require('../templates/group-list-section.html'),
+GroupListSection.propTypes = {
+  /* The list of groups to be displayed in the group list section. */
+  sectionGroups: propTypes.arrayOf(propTypes.object),
+  /* The string name of the group list section. */
+  heading: propTypes.string,
+  /* A boolean indicating whether out of scope group selection should be disabled. */
```

Comment:
I don't immediately understand what this means.

Reply:
Since there was a logic change to isScopedToUri I believe the disableOosGroupSelection is no longer needed. (You'll have to rebase to master to see that change cause it was just merged). This should be sufficient now:
```const isSelectable = !group.scopes.enforced || group.isScopedToUri```

Case 2 should never exist. We should never be direct linking to a group with an enforced scope that is out of scope.

This used to be necessary because isScopedToUri took into account enforcement.